apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js

```diff
@@ -0,0 +1,86 @@
+import { assert } from '../../../util/configuredChai';
+import {stubRedux, restoreRedux, registerReducers} from '@cdo/apps/redux';
+import manageStudents, {
+  setLoginType,
+  setSectionId,
+  setStudents,
+} from '@cdo/apps/templates/manageStudents/manageStudentsRedux';
+
+const studentEmailData = [
+  {
+    id: 1,
+    name: 'StudentNameA',
+    username: 'student1',
+    userType: 'student',
+    age: 17,
+    gender: 'f',
+    loginType: 'email',
+    secretWords: 'wizard',
+    secretPictureName: 'wizard',
+    secretPicturePath: '/wizard.jpg',
+    sectionId: 53,
+  },
+  {
+    id: 2,
+    name: 'StudentNameC',
+    username: 'student2',
+    userType: 'student',
+    age: 14,
+    gender: 'm',
+    loginType: 'email',
+    secretWords: 'wizard',
+    secretPictureName: 'wizard',
+    secretPicturePath: '/wizard.jpg',
+    sectionId: 53,
+  },
+  {
+    id: 3,
+    name: 'StudentNameD',
+    username: 'student3',
+    userType: 'student',
+    age: 9,
+    gender: 'm',
+    loginType: 'email',
+    secretWords: 'wizard',
+    secretPictureName: 'wizard',
+    secretPicturePath: '/wizard.jpg',
+    sectionId: 53,
+  }
+];
+
+describe('manageStudentsRedux', () => {
+  const initialState = manageStudents(undefined, {});
+
+  beforeEach(() => {
+    stubRedux();
+    registerReducers({manageStudents: manageStudents});
+  });
+
+  afterEach(() => {
+    restoreRedux();
+  });
```

Comment:
I don't think you need any of the before/after here. You're not really using redux at all - just testing the reducer directly.